app/routes.py

```diff
@@ -1,2 +1,184 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, make_response, request
+from app.models.task import Task 
+from app.models.goal import Goal
+from app import db
+from datetime import date
+import requests
+import os
+from dotenv import load_dotenv
 
+load_dotenv()
+
+task_bp = Blueprint("task_bp", __name__, url_prefix="/tasks")
+goal_bp = Blueprint("goal_bp", __name__, url_prefix="/goals")
+
+@task_bp.route("", methods=["GET", "POST"])
+def handle_all_tasks():
+    tasks_response = []
+    
+    #WV2: Get Tasks Asc, Get Tasks Desc
+    if request.method == "GET":
+        if request.args.get('sort') == 'asc':
+            tasks = Task.query.order_by(Task.title.asc()).all()
+        elif request.args.get('sort') == 'desc':
+            tasks = Task.query.order_by(Task.title.desc()).all()
+
+        #WV1: No Saved Tasks, One Saved Tasks
+        else:
+            tasks = Task.query.all()     
+        for task in tasks:
+            tasks_response.append(task.to_dict())
+        return jsonify(tasks_response), 200 
+
+    #WV1: Create Task With None Completed_At, Create Task - Title, Create Task - Description, Create Task - Completed_At
+    #WV3: Create Task Wtih Valid Completed_At
+    if request.method == "POST":
+        request_body = request.get_json()
+        try:
+            new_task = Task(title = request_body["title"], 
+                            description = request_body["description"],
+                            completed_at = request_body["completed_at"])
+        except KeyError:
+            return ({'details': 'Invalid data'}, 400)        
```

Comment:
Cool use of try and except! I especially like that you specify that `KeyError` is the only type of exception that should be caught.